changelog/unreleased/fix-last-month-search.md

```diff
@@ -0,0 +1,6 @@
+Bugfix: Fix last month search
+
+Fix the last month edge case when currently is 31-th https://github.com/golang/go/issues/31145
```

Comment:
... for the last day in months with 31 days

also please move the issue to the footnote to support calens changelog gen